src/main/java/Game.java

```diff
@@ -25,4 +64,30 @@ public boolean isBoardFull() {
         return true;
     }
 
+    public boolean isGameWon(List<String[]> lines) {
+        for (String[] line : lines) {
+            if (allAreX(line) || allAreO(line)) {
```

Comment:
The functions allAreX and allAreO are very similar. One thing to consider is a more general function that takes the "X" or "O" as input. However stepping back a bit is it actually necessary to check for a specific symbol to know if a line is won?

Reply:
Also, it may be a good thing to avoid to use O into a method name as is can be quite difficult to know if it's, in fact, a 0 (zero).

Same things happen with I and l, which one is a i, which one is a L?

Reply:
Thanks both! I have made this more generalised.